### DISCUSSION TITLE
Explaining Z-normalized Euclidean Distance

### DISCUSSION THREAD

**Discussion by Sandy4321** [TDAmeritrade/stumpy#37] [Q&A]
great tutorial https://stumpy.readthedocs.io/en/latest/Tutorial_0.html
BUT may you add to tutorial explanation what z-normalized-euclidean-distance-derivation is
it is not trivial , for example 
https://stats.stackexchange.com/questions/266553/z-normalized-euclidean-distance-derivation

**Comment by seanlaw** [comment]
Thank you for your kind words. Is there something specific that you'd like to know? 

I recommend talking a look at [this video](https://youtu.be/LnQneYvg84M?t=368) to understand what the z-normalized Euclidean distance is.

I think that it is too distracting to add a long explanation of what z-normalized Euclidean distance is in the tutorial but I will add a link to the video above.

**Comment by seanlaw** [comment]
@Sandy4321 closing this for now but feel free to reopen if you have continued questions.
